docs/portal/registry-users/changing-passwords.md

```diff
@@ -0,0 +1,8 @@
+# Changing Passwords
```

Comment:
This operation requires admin portal user permission. Please add a caveat on this page.

Reply:
Done.